src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,190 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * A personal assistant chatbot.
+ */
+public class Duke {
+    private static final String INDENTED_HORIZONTAL_LINE = " ".repeat(4) + "_".repeat(60);
+    /** Platform independent line separator */
+    private static final String LS = System.lineSeparator();
```

Comment:
Suggest to avoid short-form variable names; replace the name with a more verbose one.

Reply:
Done, changed to `LINE_SEPARATOR`.